roles/netbox/defaults/main.yml

```diff
@@ -0,0 +1,136 @@
+---
+netbox_path_base: /opt/netbox
+
+netbox_version: "3.7.5"
+
+netbox_user_name: netbox
+
+netbox_fqdn: "https://{{ netbox_domain }}"
+netbox_allowed_hosts:
+  - "{{ netbox_domain }}"
+
+netbox_login_required: true
+
+# Extra Plugins to install
+netbox_plugins:
+  - 'netbox_topology_views'
+  - 'netbox_inventory'
+  - 'netbox_bgp'
+
+# Quality of Life
+netbox_protection_rules:
+  "dcim.site":
+    - "status":
+        - "eq": "decommisioning"
+
+netbox_path_yaml_config: "{{ netbox_path_base }}/config"
+netbox_path_state: "{{ netbox_path_base }}/state"
+netbox_path_static: "{{ netbox_path_base }}/netbox/static"
+netbox_path_socks: "{{ netbox_path_base }}/socks"
+netbox_path_socks_uwsgi: "{{ netbox_path_socks }}/uwsgi.sock"
+netbox_path_nginx_config: "{{ netbox_path_base }}/nginx.conf"
+netbox_path_uwsgi_config: "{{ netbox_path_base }}/uwsgi.ini"
+netbox_file_config_py: "{{ netbox_path_base }}/netbox/netbox/configuration.py"
+
+netbox_container_port: 8000
+
+netbox_container_name: "netbox"
+netbox_container_env: {}
+netbox_container_ports: []
+netbox_container_labels_merged: >-
+  {{ netbox_container_labels_base |
+  combine(netbox_container_labels | default({})) }}
+netbox_container_labels_base:
+  version: "{{ netbox_version }}"
+netbox_container_volumes_common:
+  - "{{ netbox_path_base ~ '/configuration.py' ~ ':' ~ netbox_file_config_py }}"
+  - "{{ netbox_path_yaml_config ~ ':' ~ netbox_path_yaml_config }}"
+  - "{{ netbox_path_state ~ ':' ~ netbox_path_state }}"
+  - "{{ netbox_postgres_socket ~ ':' ~ netbox_postgres_socket }}"
+  - "{{ netbox_path_static ~ ':' ~ netbox_path_static }}"
+  - "{{ netbox_path_socks ~ ':' ~ netbox_path_socks }}"
+  - "{{ netbox_path_uwsgi_config ~ ':' ~ netbox_path_uwsgi_config }}"
+  - "/etc/passwd:/etc/passwd:ro"
+
+netbox_container_volumes_nginx:
+  - "{{ netbox_path_nginx_config }}:/etc/nginx/conf.d/netbox.conf"
+  - "{{ netbox_path_state ~ ':' ~ netbox_path_state }}"
+  - "{{ netbox_path_static ~ ':' ~ netbox_path_static }}"
+  - "{{ netbox_path_socks ~ ':' ~ netbox_path_socks }}"
+
+netbox_container_restart_policy: "always"
+netbox_container_image_state: "present"
+netbox_container_image_force_pull: true
+netbox_container_image_reference: >-
+  {{
+    netbox_container_image_repository
+    ~ ":"
+    ~ netbox_container_image_tag | default('v' ~ netbox_version ~ '-r0')
```

Comment:
Why is the release hardcoded? Also, we use the `r` notation in alpine based images because that's how they name their builds - if we do it ourselves, we should probably use [build metadata](https://semver.org/#spec-item-10) instead

Reply:
I think we can also just tag the plain version since it's barely anything.
for having metadata tags I would say open an issue in https://github.com/famedly/container-image-netbox ?